authentik/sources/oauth/types/azure_ad.py

```diff
@@ -61,3 +50,11 @@ class AzureADType(SourceType):
         "https://login.microsoftonline.com/common/.well-known/openid-configuration"
     )
     oidc_jwks_url = "https://login.microsoftonline.com/common/discovery/keys"
+
```

Comment:
For this we should probably support groups, but in a way that's compatible with the Entra source (even though not directly related)